prisma/schema.prisma

```diff
@@ -20,12 +20,22 @@ model User {
 }
 
 model ShoppingList {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
-  title     String   @db.VarChar(255)
-  finished  Boolean  @default(false)
-  user      User     @relation(fields: [userId], references: [id])
-  userId    String
+  id                   Int                    @id @default(autoincrement())
+  createdAt            DateTime               @default(now())
+  title                String                 @db.VarChar(255)
+  finished             Boolean                @default(false)
```

Comment:
Recomendo usar timestamps ao invés de Boolean flags para essa finalidade.

A ideia é ao invés de ter uma propriedade `finished=false` ou `finished=true`, vc teria `finishedAt=null` ou `finishedAt=2022-08-18T13:04:00.000-03`, respectivamente.

Com base na proposta do sistema, acredito que seria relevante ter a informação de quando a shopping list foi completada. Usando timestamps, vc consegue juntar as duas informações (quando é se foi completada) usando uma única propriedade.

Reply:
Esse campo `finished` é apenas pra saber se o usuário já utilizou aquela lista, por isso de ele ser boolean. 
É mais para o usuário ter controlle das listas ativas e inativas

Reply:
Tranquilo. Quando for relevante saber quando a lista foi concluída, aí já vai ser mais interessante fazer esse uso timestamps/null